.github/workflows/oneapi.yml

```diff
@@ -99,14 +100,12 @@ jobs:
       - name: Run DOLFINx demos (Python, MPI (np=2))
         run: |
           . /opt/intel/oneapi/setvars.sh
-          pytest -v -m mpi --num-proc=2 python/demo/test.py
+          pytest -m mpi --num-proc=2 python/demo/test.py
       - name: Run DOLFINx Python unit tests (serial)
         run: |
           . /opt/intel/oneapi/setvars.sh
-          cd python/test
-          pytest -n=auto --durations=50 unit/
+          pytest -vs -n=auto --durations=50 python/test/unit/fem
```

Comment:
Will limiting the tests to `fem` be reverted before merge?

(similarly for the mpi case below)